### DISCUSSION TITLE
Whether I can control the position and posture of HoloLens2 camera without being affected by my head.

### DISCUSSION THREAD

**Discussion by shoukailiang** [microsoft/MixedRealityToolkit-Unity#11413] [Q&A]
Whether I can control the position and posture of HoloLens2 camera（Virtual camera） without being affected by my head.

**Comment by AMollis** [comment]
@shoukailiang, thank you for your issue.  This sounds like a general question on HoloLens development, independent of MRTK functionality.  Could you post your question to the discussion forum and someone there will be able to help answer it.

**Comment by keveleigh** [comment]
With legacy XR, I'm not sure of a way to do that.
With Windows XR Plugin or OpenXR, you can just make sure there's no Tracked Pose Driver on the camera game object. In MRTK2, this could get added automatically at runtime, so you may need to disable the camera system. In MRTK3, you should just need to remove it from the prefab instance (if using the MRTK XR Rig).

**Comment by keveleigh** [comment]
> Could you post your question to the discussion forum and someone there will be able to help answer it.

@AMollis, by "discussion forum" are you referring to the Discussions tab on this repo on GitHub? If so, you can convert this issue directly! https://docs.github.com/en/discussions/managing-discussions-for-your-community/moderating-discussions#converting-an-issue-to-a-discussion

**Reply by AMollis** [reply]
@keveleigh, thank you!

**Comment by shoukailiang** [comment]
Thanks a lot.

**Comment by shoukailiang** [comment]
When I manually control the camera in HoloLens2（i disabled the Tracked Pose Driver Script）, it seems like the program's interaction is disabled. Is there a reason for this, or is it just how it works?
